docs/source/outputs.rst

```diff
@@ -22,6 +22,7 @@ Out of the box, StreamAlert supports:
 * **PagerDuty**
 * **Phantom**
 * **Slack**
+* **Microsoft Teams**
```

Comment:
Can you also update service type `teams` to line 41 on this page?

Reply:
consider it done :)